test/go-tests/k8s_utils.go

```diff
@@ -159,3 +160,37 @@ func WaitForDeploymentInNamespace(deploymentName, namespace string) error {
 func WaitForPodOfDeployment(deploymentName string) error {
 	return keptnkubeutils.WaitForDeploymentToBeRolledOut(false, deploymentName, GetKeptnNameSpaceFromEnv())
 }
+
+type K8SEvent struct {
+	Reason string `json:"reason"`
+	Type   string `json:"type"`
```

Comment:
would it make sense also to include the message?

Reply:
done